tools/manifest/manifest.py

```diff
@@ -471,6 +470,12 @@ def from_json(cls, tests_root, obj, types=None):
         self._path_hash = {to_os_path(k): v for k, v in iteritems(obj["paths"])}
 
         for test_type, type_paths in iteritems(obj["items"]):
+            # Drop "stub" items, which are no longer supported but may be
+            # present when doing an incremental manifest update.
+            # See https://github.com/web-platform-tests/rfcs/pull/27 for background.
+            if test_type == "stub":
+                continue
```

Comment:
Does this result in an incremental update being different to a rebuild? Or do we then add all of them to support when updating?

Reply:
This just tolerates the and ignores the entry if it exists, but it'll still be dropped when writing a new manifest.

As it turns out, #18745 caused the "stub" entry that this is accepting to be removed, it's not just an empty array. Compare:
https://github.com/web-platform-tests/wpt/releases/download/merge_pr_18690/MANIFEST-ddaab2cf6795cbfa77efdae981b8c494b47e7518.json.bz2
https://github.com/web-platform-tests/wpt/releases/download/merge_pr_18745/MANIFEST-9d89fa83724d2dd6a1cd82975934ff55b8770b42.json.bz2

Confirmed that incremental and full rebuild give the same results.

Reply:
> Confirmed that incremental and full rebuild give the same results.

That's all I wanted to be sure of!